src/content/docs/en/editor-setup.mdx

```diff
@@ -116,7 +116,26 @@ To add support for formatting `.astro` files outside of the editor (e.g. CLI) or
     }
     ```
 
-3. Run the following command in your terminal to format your files.
+3. Optionally, install other Prettier plugins for your project, and add them to the configuration file. These additional plugins may need to be listed in a specific order. For example, if you use Tailwind, `prettier-plugin-tailwindcss` must be [the last Prettier plugin in the plugins array](https://github.com/tailwindlabs/prettier-plugin-tailwindcss#compatibility-with-other-prettier-plugins).
+
+    ```json title=".prettierrc"
+    {
+      "plugins": [
+        "prettier-plugin-astro",
+        "prettier-plugin-tailwindcss" // needs to appear after other Prettier plugins
```

Comment:
```suggestion
        "prettier-plugin-tailwindcss" // needs to be last
```
Something shorter here would avoid any horizontal scroll on the code snippet!

If this is a prettier file, and this is the `plugins` section, is there anything *other than* a Prettier plugin that could be listed here? Maybe we don't need to be so specific?